module/zfs/vdev.c

```diff
@@ -3678,7 +3678,9 @@ EXPORT_SYMBOL(vdev_offline);
 EXPORT_SYMBOL(vdev_clear);
 
 module_param(metaslabs_per_vdev, int, 0644);
+/* BEGIN CSTYLED */
```

Comment:
It probably makes sense to wrap the whole module parameter block in these cases. That'll make it easier if another parameter is added with the same issue.